pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -463,6 +471,12 @@ func (s *Service) SearchUsersPermissions(ctx context.Context, usr identity.Reque
 		return nil, err
 	}
 
+	if s.features.IsEnabled(ctx, featuremgmt.FlagAccessActionSets) {
```

Comment:
Note: If we ever change basic roles to be composed of action sets, we will have to remember that we need to update search.